content/en/xlt/release-notes/8_4_x.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 8.4.x
+linkTitle: 8.4.x
+
+weight: 64
+type: docs
+
+date: 2024-08-05
+
+description: >
+    Export errors and events via OpenTelemetry
+
+sitemap:
+  changefreq: never
+  priority: 0.1
+---
+
+## XLT 8.4.0
+
+### Load Testing
+
+#### Export Errors and Events via OpenTelemetry
+
+XLT can now be configured to report error and event data to an external system while the load test is still running. This allows you to analyze errors and events as they occur without having to download the full result data and to create a load test report.
+
+XLT uses [OpenTelemetry](https://opentelemetry.io/) to export this data, specifically via OpenTelemetry's *Logs* interface. As the receiving system, we recommend using an [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/). Configure this collector as needed to further process and export the data to your actual analytics backend, such as Google Cloud Logs.
+
+If you want to use this feature, see below for what you need to configure in your load test suite:
+
+```properties
+### Whether OpenTelemetry real-time reporting is enabled.
+xlt.reporting.otel.enabled = true
+
+### Log Record Exporter
+otel.logs.exporter = otlp
+otel.exporter.otlp.endpoint = http://localhost:4318
+otel.exporter.otlp.protocol = http/protobuf
+## Custom headers
+otel.exporter.otlp.headers =
+
+### Batch Log Record Processor
+#otel.blrp.schedule.delay = 2000
+#otel.blrp.max.queue.size = 2048
+#otel.blrp.max.export.batch.size = 512
+
+## Resource attributes
+otel.resource.attributes = project=MyShop,loadTest=42
+```
+
+After enabling the feature, you need to configure how to reach your OpenTelemetry collector. You can then define custom headers, e.g. with authorization information, etc. Optionally, you can also reconfigure the OpenTelemetry batch log record processor.
+
+Finally, define a set of custom attributes (or labels) to be attached to all sent log entries. Use these values to separate the logs of one load test from the logs of all other load tests in your analytics backend.
```

Comment:
Please refrain to use the term 'label'. Otherwise, users may get the impression that these attributes are translated to labels in their log storage automatically. 

```suggestion
Finally, define a set of custom attributes to be attached to all sent log entries. Attributes are the OpenTelemetry term for _metadata_ and add key-value information to the telemetry producing entity.
For example, you can use them to separate the log entries emitted by one load test from the log entries emitted by the other load tests in your log storage backend.
```

Reply:
Okay, killed labels. Rephrased the rest a bit.

Reply:
:+1:  Like it. Thank you.